ceremonies/src/migrations.rs

```diff
@@ -22,6 +22,52 @@ mod v0 {
 pub mod v1 {
 	use super::*;
 
+	#[derive(
+		Default, Encode, Decode, Copy, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen,
+	)]
+	pub enum Reputation {
+		// no attestations for attendance claim
+		#[default]
+		Unverified,
+		// no attestation yet but linked to reputation
+		UnverifiedReputable,
+		// verified former attendance that has not yet been linked to a new registration
+		VerifiedUnlinked,
+		// verified former attendance that has already been linked to a new registration
+		VerifiedLinked,
+	}
+
+	impl Reputation {
+		pub fn migrate_to_v2(
+			self,
+			linked_cindex: CeremonyIndexType,
+		) -> encointer_primitives::ceremonies::Reputation {
+			match self {
+				Reputation::UnverifiedReputable => {
+					encointer_primitives::ceremonies::Reputation::UnverifiedReputable
+				},
+				Reputation::VerifiedLinked => {
+					encointer_primitives::ceremonies::Reputation::VerifiedLinked(linked_cindex)
+				},
+				Reputation::VerifiedUnlinked => {
+					encointer_primitives::ceremonies::Reputation::VerifiedUnlinked
+				},
+				Reputation::Unverified => encointer_primitives::ceremonies::Reputation::Unverified,
+			}
```

Comment:
Not really necessary to change, but still more readable and less error-prone:


```rust
			match self {
				Reputation::VerifiedLinked => {
					encointer_primitives::ceremonies::Reputation::VerifiedLinked(linked_cindex)
				},
				reputation => reputation
			}
```

Reply:
wasn't aware of that. I'm surprised that this is considered type-safe by rustc

Reply:
I was right, it isn't ;-)

Reply:
`expected `encointer_primitives::ceremonies::Reputation`, found `migrations::v1::Reputation``

Reply:
well, enum variants are all the same type.